index.html

```diff
@@ -0,0 +1,526 @@
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Here’s what I got:
+                <br>• Coumadin 5mg
+                <br>• Take one tablet daily
+                <br>• Prescribed by Dr. Ruth Parker
+                <br>• Pharmacy: Rite Aid #25335
+                <br>
+              </p>
+            </div>
+            <div class="bubble bot-avatar flo" data-delay="7500">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Is that right?</p>
+            </div>
+            <div class="bubble -response" data-delay="9600">
+              <p>Yep</p>
+            </div>
+            <div class="bubble bot-avatar flo" data-delay="10800">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Ok, added</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+
+      <section class="conversation-card content-wrap -temp">
+        <div class="about-flo-icon__bg"></div>
+        <div class="about-flo-icon"></div>
+        <div class="narrow-padding about-flo">
+          <div class="-content">
+            <p class="text -medium -color-lightblue -caps -bold centered">She checks up on you</p>
+            <p class="text -small -serif">As you start taking new medications, Florence will follow up to see if you’re experiencing side effects.</p>
+            <p class="text -small -serif">She can offer additional information to help you understand if your experience is typical — or if you should talk with your doctor.</p>
+          </div>
+          <div class="-chat">
+            <div class="bubble bot-avatar flo" data-delay="0">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>It’s been three days since you started taking Lisinopril. How’re you feeling?</p>
+            </div>
+            <div class="bubble -response" data-delay="2000">
+              <p>a little weak and lightheaded</p>
+            </div>
+            <div class="bubble flo" data-delay="3800">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Feeling lightheaded is a common side effect of Lisinopril. But you should let Dr. Ruth Parker know as soon as possible</p>
+            </div>
+            <div class="bubble bot-avatar flo" data-delay="5700">
+              <div class="spinner">
+                <span></span>
+                <span></span>
+                <span></span>
+              </div>
+              <p>Here’s her number: (919) 545-2020</p>
```

Comment:
Phone number can be wrapped in an `a` tag with a `tel` attr, I have to double check what the best practice is but keep in mind on a smart phone these will show up as links and we need to account for that.